README.md

```diff
@@ -4,21 +4,24 @@ This walkthrough illustrates how to adopt TypeScript in an existing React/Babel/
 
 If you are starting a new React project instead of converting one, you can use [this tutorial](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/tutorials/React.md).
 
-Adopting TypeScript in any project can be broken down into 2 phases,
- * Adding TypeScript compiler (tsc) to your build pipeline.
- * Converting JavaScript files into TypeScript files.
+Adopting TypeScript in any project can be broken down into 2 phases:
 
-# Understand the existing JavaScript project
+* Adding TypeScript compiler (tsc) to your build pipeline.
+* Converting JavaScript files into TypeScript files.
+
+## Understand the existing JavaScript project
 
 Before we dive into TypeScript adoption, let's take a look at the structure of the TicTacToe app. It contains a few components and looks like below with or without TypeScript.
 
+```html

```

Comment:
This isn't supposed to be a code block. It's supposed to actually be html to render a centered image in the resulting page.

Reply:
thx!